pkg/machine/vmconfigs/machine.go

```diff
@@ -0,0 +1,200 @@
+package vmconfigs
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	define2 "github.com/containers/podman/v4/libpod/define"
+	"github.com/containers/podman/v4/pkg/machine/define"
+	"github.com/containers/podman/v4/pkg/machine/lock"
+	"github.com/containers/podman/v4/utils"
+)
+
+/*
+  info        Display machine host info					common
+  init        Initialize a virtual machine				specific
+  inspect     Inspect an existing machine				specific
+  list        List machines								specific
+  os          Manage a Podman virtual machine's OS		common
+  rm          Remove an existing machine				specific
+  set         Set a virtual machine setting				specific
+  ssh         SSH into an existing machine				common
+  start       Start an existing machine					specific
+  stop        Stop an existing machine					specific
+*/
+
+var (
+	SSHRemoteConnection     RemoteConnectionType = "ssh"
+	DefaultIgnitionUserName                      = "core"
+	ForwarderBinaryName                          = "gvproxy"
```

Comment:
`DefaultIgnitionUserName` is already defined in `pkg/machine/config.go` and in the `ignition package`. Can we find a common place? maybe define somewhere? `ForwarderBinaryName` is also already defined in `pkg/machine/config.go`